fmriprep/workflows/bold/base.py

```diff
@@ -542,10 +518,10 @@ def init_func_preproc_wf(bold_file, ignore, freesurfer,
                                            mem_gb=mem_gb['filesize'],
                                            omp_nthreads=omp_nthreads)
 
-            subset_reg_reports = pe.JoinNode(niu.Select(index=0),
-                                             name='subset_reg_reports',
-                                             joinsource=inputnode,
-                                             joinfield=['inlist'])
+            # subset_reg_reports = pe.JoinNode(niu.Select(index=0),
```

Comment:
Temporary patches while I review the multiecho path

Reply:
What's the story with multiecho? Was it merged prematurely, or is it just related to this PR? (I haven't been paying much attention.)

If the latter, does it make sense to hold this PR until that's resolved?

Reply:
This particular joinnode can be removed: it merged the 3 echos in only one EPI-to-T1w reportlet. Since now that reportlet is generated within the workflow (and one of the features added in that PR is joining the echoes), then only one reportlet is added. When the `--t2-coreg` option is not set, it naturally generates one registration plot per echo (which we may want to change and register only the first echo, but that was outside the scope of that PR).

The rest of commented out comment refers to the ROIs plot, which will be revisited with the upcoming implementation of MEICA.